internal/sidekick/internal/language/walk_templates_dir_test.go

```diff
@@ -23,6 +24,9 @@ import (
 func TestWalkDir(t *testing.T) {
 	// It should get the `*.md.mustache` files and skip `partial.mustache`
 	got := WalkTemplatesDir(templates, "testTemplates")
+	for i := range got {
+		got[i].OutputPath = strings.ReplaceAll(got[i].OutputPath, "\\", "/")
```

Comment:
shouldn't we use `ToSlash()` here too?  If there is a reason why that won't work (types?) then maybe a comment?